src/test/java/bvn/car/TestCar.java

```diff
@@ -0,0 +1,103 @@
+package bvn.car;
+
+import bvn.car.model.Car;
+import bvn.car.model.Door;
+import bvn.car.model.Passenger;
+import bvn.car.model.Tire;
+import bvn.car.model.Wheel;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestCar {
```

Comment:
```suggestion
public class CarTest {
```
See: https://mate-academy.github.io/style-guides/java/java.html#s5.2.2-class-names

Reply:
changed